src/client/sandbox/event/message.js

```diff
@@ -196,8 +195,17 @@ export default class MessageSandbox extends SandboxBase {
         var message = MessageSandbox._wrapMessage(MESSAGE_TYPE.service, msg);
 
         // NOTE: For iframes without src.
-        if (!this._isIframeRemoved() && (isIframeWithoutSrc || !isCrossDomainWindows(targetWindow, this.window) &&
-                                                               targetWindow[this.RECEIVE_MSG_FN])) {
+        if (isIframeWithoutSrc || !isCrossDomainWindows(targetWindow, this.window) && targetWindow[this.RECEIVE_MSG_FN]) {
+            if (this.isWindowUnloaded) {
+                targetWindow[this.RECEIVE_MSG_FN]({
+                    // NOTE: Cloning a message to prevent this modification.
+                    data:   parseJSON(stringifyJSON(message)),
+                    source: this.window
+                });
```

Comment:
```javascript
targetWindow[this.RECEIVE_MSG_FN]({
                     // NOTE: Cloning a message to prevent this modification.
                     data:   parseJSON(stringifyJSON(message)),
                     source: this.window
                 });
```
duplicate code